packages/block-library/src/navigation/index.php

```diff
@@ -980,6 +980,8 @@ function block_core_navigation_get_fallback_blocks() {
 	if ( class_exists( 'WP_Navigation_Fallback' ) ) {
 		$navigation_post = WP_Navigation_Fallback::get_fallback();
 	} else {
+		// Refactoring required: this class must be guarded as it is available only in Gutenberg, and not in Core.
+		// phpcs:ignore Gutenberg.CodeAnalysis.ForbiddenFunctionsAndClasses.ForbiddenClassUsage
```

Comment:
@tjcafferkey @ockham Did we introduce the called to the `Gutenberg_` prefixed version of this class in the recent work on hooks in the Nav block?

Reply:
I'm removing this in https://github.com/WordPress/gutenberg/pull/58369 I don't see where this class is defined at all.